blobs.go

```diff
@@ -92,6 +92,10 @@ type BlobStatter interface {
 	Stat(ctx context.Context, dgst digest.Digest) (Descriptor, error)
 }
 
+type BlobAccessFactory interface {
+	BlobAccessController(svc BlobDescriptorService) BlobDescriptorService
```

Comment:
Since we can't completely replace the controller, we should use the original methods. For example [Clear](https://github.com/docker/distribution/blob/master/registry/storage/linkedblobstore.go#L425) is used to [delete](https://github.com/docker/distribution/blob/master/registry/storage/linkedblobstore.go#L224) blobs.